lib/utils/remote-build.ts

```diff
@@ -405,8 +405,10 @@ async function getRemoteBuildStream(
 	// We only show the spinner when outputting to a tty
 	if (process.stdout.isTTY) {
 		const visuals = getVisuals();
+		const uploadUrl =
+			process.env.BALENARC_BUILDER_URL || `https://builder.${build.baseUrl}`;
```

Comment:
Minor point: this duplicates the URL forming logic from the `getBuilderEndpoint` function, such that if someone changed either of them, they'd have to remember to change it in the other place too. A short alternative that comes to mind would be:
```js
    `Uploading source package to ${new URL(builderUrl).origin}`
```
Just a suggestion, no worries if you prefer to do it as you've coded it.

Reply:
What you say makes perfect sense - such repetition is undesirable at best. The snippet you've suggested looks nice and I think I will go with it. Thanks much!
Do you think this change deserves adding a test case(s)? Willing to do that if required.

Reply:
No test case required for this change in my opinion, but test it manually just to confirm that the output is still as expected.